src/store.ts

```diff
@@ -117,16 +119,29 @@ state
     switch (manifest?.sdk()) {
       case 'rust':
         {
-          manifest
-            .overrideWorkspaceConfig(
-              'rust-analyzer',
-              'server.path',
-              'rust-analyzer'
-            )
-            .then(
-              () => { }, // eslint-disable-line @typescript-eslint/no-empty-function
-              () => { } // eslint-disable-line @typescript-eslint/no-empty-function
-            )
+          if (vscode.workspace.getConfiguration(`${EXT_ID}`).get('enableRustAnalyzerServerPathOverride') == true) {
```

Comment:
```suggestion
          if (vscode.workspace.getConfiguration(`${EXT_ID}`).get('enableRustAnalyzerServerPathOverride') === true) {
```
but also I would make the various possible  configurations an enum so you can do `.get(Config.PathOverride)` or something like that